monthly-contributions/src/test/scala/com/gu/support/workers/EndToEndSpec.scala

```diff
@@ -11,10 +11,12 @@ import com.gu.test.tags.annotations.IntegrationTest
 import com.gu.zuora.encoding.CustomCodecs.{jsonWrapperDecoder, jsonWrapperEncoder}
 import io.circe.parser._
 
+import scala.concurrent.ExecutionContext
 import scala.io.Source
 
 @IntegrationTest
-class EndToEndSpec extends LambdaSpec {
+class EndToEndSpec(implicit ec: ExecutionContext) extends LambdaSpec {
```

Comment:
I don't think this is needed, can you try without it?

Reply:
It doesn't work without